src/PowerShellEditorServices/Services/PowerShell/Host/PsesInternalHost.cs

```diff
@@ -769,8 +789,13 @@ private void RunExecutionLoop(bool isForDebug = false)
                 {
                     try
                     {
-                        task.ExecuteSynchronously(cancellationScope.CancellationToken);
+                        ExecuteTaskSynchronously(task, cancellationScope.CancellationToken);
                     }
+                    // Our flaky extension command test seems to be such because sometimes another
```

Comment:
I put this back and documented it. It hasn't _caused_ any issues yet, and can't be put inside `ExecuteTaskSynchronously`.